### DISCUSSION TITLE
Как корректно обновлять на новую версию через LUCI?

### DISCUSSION THREAD

**Discussion by planbo1** [remittor/zapret-openwrt#28] [Q&A]
Вопрос к установке по инструкции https://github.com/remittor/zapret-openwrt/wiki/Installing-zapret%E2%80%90openwrt-package
Нужно ли сначала удалять старые пакеты, а лишь затем ставить новую версию или можно ставить поверх. 
Если нужно удалять, то нужно как-нибудь дополнительно подчищать перед установкой нового? ` /opt/zapret/uninstall_easy.sh` к примеру.
И как быть с конфигом/настройками в этом случае? Как их сохранять/восстанавливать?

**Comment by remittor** [comment]
Просто устанавливаете новый ipk/apk и всё. Там внутри в коде всё заранее учтено. Конфиги не обновятся (кроме новых опций).

**Reply by Encymos** [reply]
Уточнение:
А накатить только Luci на официальный дистрибутив запрета можно, или нужно официалку сносить и переустанавливать отсюда?

Обновил на 69 версию, поставил с 0 оф версию, но люси не хватает(

**Reply by alex30gg** [reply]
не нашел описания изменений... что нового в новой версии ? у меня стоит 69
